_posts/2025-07-27-5.13.0-released.md

```diff
@@ -0,0 +1,74 @@
+
+## Working with Nodes
+
+Here's a handful of methods you can use when working with nodes and chunks.
+
+* `core.get_node_raw()` to query nodes from the map, but faster, and with a content ID ([#16265](https://github.com/luanti-org/luanti/pull/16265))
+* `VoxelManip:initialize()` to create an empty VM storage object ([aa1bab2](https://github.com/luanti-org/luanti/commit/aa1bab21564cdac01ed2eaed6c2284fc1497626a))
+* `core.get_mapgen_chunksize()` to determine the configured chunksize for your custom mapgens ([#16289](https://github.com/luanti-org/luanti/pull/16289))
+
+Handy, no?
+
+## Show Them Their Inventory
+
+It might not seem significant, but manually pulling up a player's inventory has been a long-requested feature. Now, by passing an empty string (`""`, reserved for the player inventory) as the formname to `core.show_formspec()`, you can open the player's inventory without their input ([#15963](https://github.com/luanti-org/luanti/pull/15963)). I bet this could be used in some creative ways...
+
+## A Couple of Controls
+
+Firstly, you can now cycle through inventory tabs using `ctrl+tab` (`+shift` to cycle in reverse). This is a standard keybind for application tab switching, so we are happy to see it arrive in Luanti ([#16167](https://github.com/luanti-org/luanti/pull/16167)).
+
+Secondly, the keybind modifier for exiting to the main menu is configurable ([#16250](https://github.com/luanti-org/luanti/pull/16250)). `ESC` plus whatever you desire. More importantly, it is no longer set by default, preventing players from accidentally leaving the game due to pressing `Shift+ESC` during quick movements. Now you can play CTF in (relative) peace.
+
+## Bugs-B-Gone
+
+The poor insects are being squashed left and right! You may be happy to learn that a plethora of issues have been resolved regarding:
+
+* 9-slice image button rendering ([#16146](https://github.com/luanti-org/luanti/pull/16146))
+* Animated world-aligned texture rendering ([#16091](https://github.com/luanti-org/luanti/pull/16091))
+* Broken keybinds ([#16049](https://github.com/luanti-org/luanti/pull/16049))
+* Lagging bone attachments and animation blending ([#14722](https://github.com/luanti-org/luanti/pull/15722))
+* Menu freezing on world join ([#16303](https://github.com/luanti-org/luanti/pull/16303))
+* `[verticalframe` transparency ([158bfa6](https://github.com/luanti-org/luanti/commit/158bfa6442b00b791161291aa8d36c6a6746d4a6))
+* Particlespawner crashes ([#16351](https://github.com/luanti-org/luanti/pull/16351))
+
+## What's Next?
+
+Back to the 3-month intervals. By the time 5.14.0 lands, you might see [camera color effects](https://github.com/luanti-org/luanti/pull/16260), [matrix and rotation APIs](https://github.com/luanti-org/luanti/pull/16212), [glTF morph animations](https://github.com/luanti-org/luanti/pull/16096), [animated inventory images](https://github.com/luanti-org/luanti/pull/15979), and [more](https://github.com/luanti-org/luanti/milestone/30)!
```

Comment:
This is nice, it gives people the idea that more tangible things are in the works